backend/src/admin/admin.service.ts

```diff
@@ -16,6 +18,10 @@ export class AdminService {
 
     register = async (registrationData: RegisterAdminDto) => this.authenticationService.registerAdmin(registrationData);
     login = async (loginData: LoginAdminDto) => this.authenticationService.loginAdmin(loginData);
+    registerJunior = async (registrationData: RegisterJuniorDto) => this.authenticationService.registerJunior(registrationData);
+
+    // This will be handed to a guard once a clear workflow is provided for admin login.
+    verifyIsAdmin = async (email: string) => { if (!(await this.getUser(email))) { throw new UnauthorizedException(content.NotAnAdmin); } };
```

Comment:
I would personally prefer to use ES6 style methods (`register`, `login`, `verifyIsAdmin`)